packages/calcite-components/src/components/action/action.tsx

```diff
@@ -59,10 +57,26 @@ export class Action extends LitElement implements InteractiveComponent, FormOwne
 
   private focusSetter = useSetFocus<this>()(this);
 
+  private indicatorEl?: HTMLDivElement;
+
   //#endregion
 
   //#region Public Properties
 
+  /**
+   * Use this property to override or extend ARIA properties and attributes on the component's button.
+   */
+  @property() aria?: Pick<
```

Comment:
This deviates from the proposed structure in an [earlier comment](https://github.com/Esri/calcite-design-system/pull/12579#discussion_r2331585621). I'd like for this property to avoid redundant naming. To address the related issue, would the following path work? cc @geospatialem 

1. Update this PR as a `fix`  
2. Make the `aria` prop internal  
3. Include this in a v3.3 patch
4. Create follow-up issue to introduce `aria` prop
5. Refactor the `aria` object to avoid redundant naming  
6. Target ☝️ as a `feat` in v4 or after?

Reply:
That sounds good to me. What's the difference between `@internal` and `@private`? @geospatialem @jcfranco

Reply:
We can use #10706 as the issue for the `feat`